podman.spec.rpkg

```diff
@@ -84,6 +84,7 @@ Requires: iptables
 Requires: nftables
 Recommends: catatonit
 Suggests: qemu-user-static
+Conflicts: quadlet
```

Comment:
We should also have a `Provides: %{name}-quadlet` here if we are removing the subpackage. An obsoletes would also be nice, but I gotta figure out the last build that shipped a separate podman-quadlet.

Reply:
Let me disable podman autobuilds and get you the last NVR to add here.

Reply:
Add these as well.
```
Obsoletes: %{name}-quadlet <= 101:0.0.git.17877.f247b4d4-1
Provides: %{name}-quadlet = %{epoch}:%{version}-%{release}
```

I will resume podman autobuilds after this merges.

Reply:
@rhatdan could you add these please? That should keep upgrades smooth.

Reply:
It has been addressed?  See Below?

Reply:
ah, my bad. LGTM